windowsforms/Sudoku/VB/sudoku/sudoku.vbproj

```diff
@@ -0,0 +1,60 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Sudoku</RootNamespace>
+    <StartupObject>Sudoku.Form1</StartupObject>
+    <UseWindowsForms>true</UseWindowsForms>
+    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
+  </PropertyGroup>
+  <PropertyGroup>
+    <LangVersion>16</LangVersion>
+    <AssemblyName>sudoku</AssemblyName>
+    <ApplicationManifest>My Project\app.manifest</ApplicationManifest>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <OptionInfer>Off</OptionInfer>
+    <OptionStrict>On</OptionStrict>
+    <Version>1.0.1</Version>
+    <ApplicationIcon>suboku_icon.ico</ApplicationIcon>
```

Comment:
Typo, rename the file and change it here.

Reply:
done